src/main/java/TechJobs.java

```diff
@@ -112,14 +117,66 @@ private static String getUserSelection(String menuHeader, HashMap<String, String
                 validChoice = true;
             }
 
-        } while(!validChoice);
+        } while (!validChoice);
 
         return choiceKeys[choiceIdx];
     }
 
     // Print a list of jobs
     private static void printJobs(ArrayList<HashMap<String, String>> someJobs) {
 
-        System.out.println("printJobs is not implemented yet");
+        if(someJobs.size()==0){
+            System.out.println("No Results");
+        }
+
+        for (int i=0; i< someJobs.size();i++){
+
+                      HashMap<String, String> TotalKeys= new HashMap<>(someJobs.get(i));
+                      String jobInfo= "";
+
+                     jobInfo+="\n*****\n";
+
+                      for(Map.Entry<String,String> JobDisplay: TotalKeys.entrySet()){
+                          jobInfo += (JobDisplay.getKey()+": "+ JobDisplay.getValue()+ "\n");
+                      }
+                      jobInfo+="*****";
+                      System.out.println(jobInfo);
```

Comment:
When you use System.out.println() it adds an extra empty line before it. This is causing tests to fail. Use System.out.print() instead